src/AppBundle/Command/ListUsersCommand.php

```diff
@@ -34,12 +34,20 @@
  *
  * @author Javier Eguiluz <javier.eguiluz@gmail.com>
  */
-class ListUsersCommand extends ContainerAwareCommand
+class ListUsersCommand extends Command
 {
-    /**
-     * @var ObjectManager
-     */
     private $entityManager;
+    private $mailer;
+    private $emailSender;
```

Comment:
I'd add `@param string` annotation since `$emailSender` does not have type hint and can't be resolved by IDE.